inc/features/blocks/namespace.php

```diff
@@ -186,7 +189,29 @@ function sanitize_id( $param ) : string {
  * @return boolean
  */
 function check_views_permission() : bool {
-	return current_user_can( 'edit_posts' );
+	return current_user_can( 'view_xb_analytics' );
```

Comment:
Should this also be in the post type registration?

Reply:
Why would it be there? This is a custom endpoint for reading analytics data for an XB, there's no post type (yet).

Reply:
I was figuring that if you can read the audience, you'd be able to read the analytics data for it.